openpype/hosts/houdini/plugins/create/create_bgeo.py

```diff
@@ -0,0 +1,90 @@
+# -*- coding: utf-8 -*-
+"""Creator plugin for creating pointcache bgeo files."""
+from openpype.hosts.houdini.api import plugin
+from openpype.pipeline import CreatedInstance
+from openpype.lib import EnumDef
+
+
+class CreateBGEO(plugin.HoudiniCreator):
+    """BGEO pointcache creator."""
+    identifier = "io.openpype.creators.houdini.bgeo"
+    label = "BGEO PointCache"
+    family = "pointcache"
```

Comment:
With us now having two creators with the same family. What does that mean for the "Convert Legacy" logic for old scenes made prior to the new publisher. Will it correctly convert to the Alembic one instead of the BGEO one?

Please make sure this still updates legacy scenes fine.

Reply:
Was this checked? Since it's marked resolved.